src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+* Where is the **API reference**?
+    * Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages. 
+* How can I get **help**?
+    * Reach out to the [Frame.io](http://frame.io/) support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
+* What are the key concepts to the Frame V4 API?
+    * V4 represents an expansion from primarily serving post-production video professionals to a discipline-agnostic platform for creatives and creative managers.  [Frame.io](http://frame.io/) acts as a central canvas for multi-disciplinary teams and solves for key needs for everything from more focus time for creative work to increased visibility into ongoing projects.
+    * **Workflow management**: Centralize, organize, and build all your end-to-end creative workflows at scale leveraging Metadata, Custom Fields, and Collections.
+    * **File transfer and management**: This set of capabilities gives users all the flexibility and power to build the best file management system for their creative process making it easier to quickly get the information they need, and manage all their work-in-progress assets at scale.
+    * **Review and approval**: [Frame.io](http://frame.io/) helps creative teams reduce the total number of revision cycles with advanced review, commenting, and annotation capabilities available across a broad set of media types.
+    * **Sharing and presentation**: Customize and share creative work from one central place.
+    * **Security and governance**: View and manage all users, workspaces, and projects in one place. Take bulk actions and export information to control how you utilize usage data.
+    * **Integrations and APIs**: Native integrations and an open API connect [Frame.io](http://frame.io/) to the systems you already use to unblock sticking points and keep work in flow.
+
+* How can I move my **Azure** **Blob** storage to [Frame.io](http://frame.io/) V4 directly?
+
+    The easiest and most efficient way to upload and download bulk content, large folder structures, and entire projects. Upload your files and folder structures directly from your desktop into your [Frame.io](http://frame.io/) projects; download the original files or choose multiple proxies. 
+
+* What is **metadata**?
+  
+  Use Metadata to tag, organize, and view creative assets based on how your team works. Tag all incoming assets with metadata to organize then group into Collections to sort and view your assets based on need (assignee, due date, approval status, etc.) for easier access and superior organization. 
```

Comment:
Not sure what Frame marketing thinks on this subject, but I think we should capitalize Metadata consistently when used in the context or maybe even go further and say "Frame.io Metadata". 

Additionally, at the moment we are referring to  Frame Metadata  as Custom Fields in our V4 API, but I think there has been some talk of changing this, so we probably need to nail this down as well.

Reply:
Took out Metadata for now.